src/components/main/Carousel.tsx

```
@@ -4,66 +4,72 @@ import { useState, useEffect } from "react"
   // 캐러셀 상품 클릭 시 상품 페이지로 이동
   const handleCardClick = () => {
-    router.push(`/project/${carouselCards[currentIndex].id}`);
+    router.push(`/project/${categoryTopProjects[currentIndex].id}`);
   }
 
-  const currentCard = carouselCards[currentIndex];
+  const currentCard = categoryTopProjects[currentIndex];
+
+  useEffect(() => {
+    loadCategoryTopProjects();
+  }, []);
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadCategoryTopProjects

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)